app/Models/CountrySport.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CountrySport extends Model
```

Comment:
@kjarya it's not necessary to define that Pivot table model.

See how I did it in my PR: #16